src/ontology/edits/oeo-physical.omn

```diff
@@ -6066,7 +6070,7 @@ pull request: https://github.com/OpenEnergyPlatform/ontology/pull/1215",
 Class: OEO_00010267
 
     Annotations: 
-        OEO_00040001 "http://openenergy-platform.org/ontology/oeo/oeo-physical",
+        OEO_00040001 "http://openenergy-platform.org/ontology/oeo/oeo-physical"^^xsd:string,
```

Comment:
That is an unexpected change and probably a mistake.

Reply:
I'd argue it is a correction of a mistake, since a) Protégé did it automatically and b) it's there in all instances
```
$ grep OEO_00040001 src/ontology/edits/*.omn | sort | uniq
src/ontology/edits/oeo-model.omn:AnnotationProperty: OEO_00040001
src/ontology/edits/oeo-model.omn:        OEO_00040001 "http://openenergy-platform.org/ontology/oeo/oeo-model"^^xsd:string,
src/ontology/edits/oeo-physical.omn:AnnotationProperty: OEO_00040001
src/ontology/edits/oeo-physical.omn:        OEO_00040001 "http://openenergy-platform.org/ontology/oeo/oeo-physical"^^xsd:string,
src/ontology/edits/oeo-shared.omn:AnnotationProperty: OEO_00040001
src/ontology/edits/oeo-shared.omn:        OEO_00040001 "http://openenergy-platform.org/ontology/oeo/oeo-shared"^^xsd:string,
src/ontology/edits/oeo-social.omn:AnnotationProperty: OEO_00040001
src/ontology/edits/oeo-social.omn:        OEO_00040001 "http://openenergy-platform.org/ontology/oeo/oeo-social"^^xsd:string,
```

Reply:
In my local oeo-physical.omn file it's also there in ~400 cases.